jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,527 @@
+      }
+    }' https://circleci.com/api/v2/project/{VCS}/{user_name}/hello-world/pipeline
+    ```
+
+This concludes the end-to-end example of using the V2 API. For more detailed information about other endpoints you may wish to call, please refer to the [CircleCI API v2 Documentation]({{site.baseurl}}/api/v2/#circleci-api) for an overview of all endpoints currently available.
+
+# Additional API Use Cases
+
+Now that you have a general understanding of how the CircleCI API v2 service works through an end-to-end API example request and walkthrough, let's look at a few common tasks and operations you may perform on a regular basis when using the API. Whether you wish to return information about a job or project, or retrieve more detailed information about a project by reviewing its artifacts, the examples shown below should assist you in gaining a better understanding of how to make some API requests to the server so you can perform a deep dive into the specifics of your work.
+
+### Prerequisites
+
+Before trying any of the API calls in this section, make sure you have met the following prerequisites:
+
+* You have set up a GitHub or BitBucket account with a repository to use with CircleCI.
+* You have completed CircleCI onboarding and you have a project setup.
+* You have a personal API token and have been authenticated to make calls to the server.
+
+This section provides detailed information on how you can perform the following tasks and operations:
+
+* [Get project details](#get-project-details)
+* [Get job details](#get-job-details)
+* [Download artifacts](#download-artifacts)
+
+## Get Project Details
+
+You may often find that it would be helpful to retrieve information about a specific project, including the name of the organization the project belongs to, the version control system (vcs) that hosts the project, and other details. The CircleCI API enables you to return this and other information by making a single GET request to the `project/{project-slug}` endpoint by passing the `project-slug` parameter.
+
+**Note** If you would like more detailed information about a project, or simply need a refresher on the specifics of a project, please refer to the CircleCI [Projects](https://circleci.com/docs/2.0/projects/) page.
+
+### Steps
+
+Of the several project-related API endpoints available with CircleCI API v2, making a GET request to the `/project/{project-slug}`endpoint enables you to return detailed information about a specific project by passing the `project_slug` parameter with your request.
+
+**Note** In this example, please note that whenever you see curly brackets `{}`, this represents a variable that you must manually enter in the request.
+
+To return project details, perform the following steps:
+
+1. Declare the parameters you expect to receive from the API. For this GET API call, under the `parameters` key, define the `project_slug` parameter you want returned in the JSON payload in your `curl` request as follows:
```

Comment:
I can't follow what the test in this step is asking me to do - sounds like I have to define project_slug somehow but then the example doesn't show how? - "...under the `parameters key`..."

Reply:
added more clarification to how to use project slug.

Reply:
What I meant was I don't understand what is meant by "under the `parameters` key, define the `project_slug` parameter you want returned..." ... I mean I get how to define the `project_slug` but the step still isn't clear to me, (e.g what's the `parameters` key?) We could explicitly state the elements as Tyler did in his example, and use a table to show what they all are like I added in the download artifacts use case (as you suggested)? Might be clearer for the reader.